parlai/crowdsourcing/tasks/acute_eval/fast_eval.py

```diff
@@ -547,14 +547,7 @@ def analyze_results(self, args: Optional[str] = None):
         self._print_progress(f'ACUTE results saved to {self.results_path}')
 
 
-defaults = [
-    '_self_',
-    {"mephisto/blueprint": FAST_ACUTE_BLUEPRINT_TYPE},
-    {"mephisto/architect": "local"},
-    {"mephisto/provider": "mock"},
-    'conf/base_fast_acute',
-    {"conf": "example_fast_acute"},
-]
+defaults = ['_self_', {"conf": "example_fast_acute"}]
```

Comment:
hmm, there's a lint error on line 35 here because FAST_ACUTE_BLUEPRINT_TYPE is now unused. You can try just removing the import, but if you find that the import can't be removed because it triggers registering the Blueprint, you can copy how I've worked around this in another file by adding https://github.com/facebookresearch/ParlAI/blob/e04a35cff3f8f11dff0cbcde6d262a3fc0adbffc/parlai/crowdsourcing/tasks/acute_eval/analysis.py#L38